backend/src/appointment/database/schemas.py

```diff
@@ -152,15 +149,16 @@ class AvailabilityBase(BaseModel):
 
 
 class Availability(AvailabilityBase):
+    model_config = ConfigDict(from_attributes=True)
+
     id: int
     time_created: datetime | None = None
     time_updated: datetime | None = None
 
-    class Config:
-        from_attributes = True
-
 
 class ScheduleBase(BaseModel):
+    model_config = ConfigDict(json_encoders = { time: lambda t: t.strftime('%H:%M') })
```

Comment:
Seems like json_encoders will be deprecated in the near future
https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.json_schema_extra

but we can punt that to a later issue.

Reply:
Good catch. I saw that but didn't find a quick way to migrate it, so let's do that later 😇